datahub-web-react/src/app/entity/dataset/profile/schema/components/SchemaHeader.tsx

```diff
@@ -44,6 +131,11 @@ type Props = {
     hasKeySchema: boolean;
     showKeySchema: boolean;
     setShowKeySchema: (show: boolean) => void;
+    lastUpdatedTimeString: string;
+    schemaBlameSelectValue: string;
+    semVerList: Array<SemVerStruct>;
+    schemaBlameView: any;
```

Comment:
We are passing the view itself as a prop? @gabe-lyons what are your thoughts?

Reply:
Also can we type this beyond "any"?

Reply:
Not a view - got it the name confused me. Really it should be typed used the enum above

Reply:
Yes lets add a type & perhaps rename to be clearer

Reply:
Agreed!